lib/metrics.js

```diff
@@ -0,0 +1,20 @@
+/* @flow weak */
+"use strict";
+var metrics = module.exports = { };
+
+var EventEmitter = require("events").EventEmitter;
+metrics.emitter = new EventEmitter();
+
+metrics.processResponse = function(request, httpCode, payload, duration) {
+  var route = request ? request.route.path : "invalid";
+  route = route.replace(/[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}/ig, ":id");
```

Comment:
Can we move the scary looking regex (which I suspect is matching UUIDs) to a helper variable to make it less scary by giving it a name?